src/plone/app/theming/browser/controlpanel.pt

```diff
@@ -611,6 +617,8 @@
                             placeholder="Put your plain css..."
                             i18n:attributes="placeholder"
                             tal:content="custom_css"
+                            class="pat-code-editor"
```

Comment:
Usually designers like `class`to be the first attribute, see:
- https://pedantic-html.readme.io/docs/attribute-order
- https://codeguide.co/#html-attribute-order